app/requirements.txt

```diff
@@ -0,0 +1,2 @@
+gradio==4.29
```

Comment:
we can add this in the main setup.py? or we want to enable to route of application development in different requirements.txt?

Reply:
depends on the type of approval we get, since we are not keeping the developed app inside the pip package, it doesn't make sense to add this in setup.py